governance/polls/SparkLend Ethereum - Remove WBTC-BTC from Kill Switch - August 26, 2024.md

```diff
@@ -0,0 +1,53 @@
+---
+title: SparkLend Ethereum - Remove WBTC/BTC from Kill Switch - August 26, 2024
+summary: Signal your support or opposition to removing WBTC/BTC from Kill Switch.
+discussion_link: https://forum.makerdao.com/t/aug-23-2024-proposal-changes-to-spark-for-upcoming-spell/24940
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-08-26T16:00:00
+end_date: 2024-08-29T16:00:00
+---
+# Poll: SparkLend Ethereum - Remove WBTC/BTC from Kill Switch - August 26, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Scope Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, August 26 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to removing the WBTC/BTC kill switch, as it is no longer necessary since new borrowings from accounts with WBTC collateral are not possible.
```

Comment:
```suggestion
The community can vote in this poll to express support or opposition to removing the WBTC/BTC kill switch, as it is no longer necessary to prevent additional borrowing in cases where WBTC depegs since new borrowings from accounts with WBTC collateral are not possible.
```